setup.py

```diff
@@ -18,9 +18,10 @@
              'scripts/neomodel_remove_labels'],
     setup_requires=[
         'pytest-runner'] if any(x in ('pytest', 'test') for x in sys.argv) else [],
-    tests_require=['pytest>=6.2.4', 'shapely', 'neobolt'],
+    tests_require=['pytest>=6.2.4'],
```

Comment:
Is it ok to remove `neobolt` from the test requirements?

Reply:
neobolt is an install requirement - so will be installed from there :) test requirements should generally be additional requirements just needed for testing e.g pytest etc..